docs/contrib/PlatformBringUp.md

```diff
@@ -0,0 +1,415 @@
+
+
+## Tapcell Configuration
+
+The tapcell configuration file is used to insert tapcells and endcaps into the design. Refer to the [Tapcell][https://github.com/The-OpenROAD-Project/OpenROAD/blob/f864efbae4ffd4cc533f0e22c57cc7ab89618c44/src/tap/README.md] documentation on how to construct this file.
+
+
+
+## setRC Configuration
+
+
+setRC allows the user to define resistances and capacitances for layers and vias using the `set_layer_rc` command. There is also a command that allows you to set the resistance and capacitance of routing wires using the `set_wire_rc`. Following is a generic example of a setRC configuration file which sets the resistance and capacitance of five metal layers, four vias, one signal wire, and one clock wire.
+
+
+```
+# setRC.tcl
+#######################
+set_layer_rc -layer M1 -capacitance 1.449e-04 -resistance 8.929e-04
+set_layer_rc -layer M2 -capacitance 1.331e-04 -resistance 8.929e-04
+set_layer_rc -layer M3 -capacitance 1.464e-04 -resistance 1.567e-04
+set_layer_rc -layer M4 -capacitance 1.297e-04 -resistance 1.567e-04
+set_layer_rc -layer M5 -capacitance 1.501e-04 -resistance 1.781e-05
+
+set_layer_rc -via V1 -resistance 9.249146E-3
+set_layer_rc -via V2 -resistance 4.5E-3
+set_layer_rc -via V3 -resistance 3.368786E-3
+set_layer_rc -via V4 -resistance 0.376635E-3
+
+set_wire_rc -signal -layer M2
+set_wire_rc -clock -layer M5
+```
+
+
+
+## KLayout
+
+KLayout is used in OpenROAD to provide GDS merging, DRC, and LVS. Two files are required for KLayout and they are generated within the KLayout GUI. Install KLayout on the host machine since it is not included in the OpenROAD build process. Then create the properties and tech files as explained below.
+
+
+## KLayout properties file
+
+Follow these steps to generate the  KLayout properties file:
+
+
+
+	1. Open KLayout
+	2. Install the tf_import package
+	    1. Inside KLayout, go to Tools
+	    2. Manage Packages
+	    3. Install New Packages
+	    4. Select tf_import
+	        1. If the source of the package is from GitHub, then the file “” needs to be edited to include “source stdio”
+	3. Re-start KLayout
+	4. File -> Import some LEF. Does not matter what LEF; you will just get an error message without one.
+		1. Once selected, go to Options at the bottom left
+		2. Select your layer map file under the Production tab
+		3. Go to the LEF+Macro Files tab, then add under Additional LEF files, the merged (original) LEF file in your platform directory
+		4. Under Macro Layout Files, add the GDS file in your platform directory
+	5. File -> Import Cadence tech file
+	    1. You have to select a tech file (found in the PDK, usually inside the Virtuoso folder)
+	    2. KLayout also needs a `.drf` file which is automatically included if it resides in the same directory the cadence tech file was found in (found in the PDK’s Virtuoso folder).
+	6. File -> Save Layer Properties
+    	1. Save as a `.lyp` file in your platform directory
```

Comment:
I don't know if these should be the primary instructions because the `.lyp` file can be created manually - it's just an XML file. In fact the `.lyp` file isn't required to obtain a GDS.

Reply:
Specified that the `.lyp` is optional and just for styling. I can place this if there is a better way of generating the `,lyp` or just remove it all together if it's unnecessary.